### ISSUE TITLE
Addon settings page can not be used properly with a keyboard or other accessibility tools

### ISSUE THREAD

**Issue by GarboMuffin** [ScratchAddons/ScratchAddons#2532]
### Description

The addon settings page can not be properly navigated with a keyboard by pressing tab. It is also probably unusable for people who rely on other accessibility tools.

To be fair, though, these users will also have a hard time using the Scratch editor or interacting with Scratch projects (which is outside the scope of this issue)

### Reproduction steps

1. Open the settings page
2. Press tab a few times

### Expected behavior

It should let you toggle addons being enabled/disabled and reset addons, at a minimum.

### Actual behavior

It does not let you toggle addons as the switches are not focusable. Similarily addons cannot be reset because the button is not focusable.

### Environment

 - Scratch Addons version: Release 1.14.3
 - Browser: any
 - Operating system: any

**Comment by WorldLanguages** [comment]
See #2877 for some discussion related to how to achieve this

We should look into this after dealing with #6272

**Comment by Samq64** [comment]
I have started working on this but am not sure how to properly handle the dropdown menus so they close when unfocusing and can be navigated with the up/down arrow keys. Maybe radio buttons could work even though the items are more like actions than selections?

**Comment by mxmou** [comment]
> I have started working on this but am not sure how to properly handle the dropdown menus so they close when unfocusing and can be navigated with the up/down arrow keys. Maybe radio buttons could work even though the items are more like actions than selections?

The W3C recommends using a normal `<ul>` and `<li>`s and writing the keyboard event handlers yourself. Ideally, they should also have the correct ARIA roles so that screen readers understand them. See https://www.w3.org/WAI/tutorials/menus/application-menus/
